### PR TITLE
#170985787 member view borrowed books

### PR DESCRIPTION
**Pull request by blaise82**
#### What does this PR do?
feat(issue): member view books he/she borrowed

#### Description of Task to be completed?
- create view borrowed function
- set route
- wrote tests

#### How should this be manually tested?
- clone this repository
- checkout to _ft-member-borrowed-book-170985787_ branch
- run `npm run install`
- run `npm run createTables`
- run npm start

`route - localhost:8080/api/v1/issue/member`
 [open in post man](https://documenter.getpostman.com/view/9143901/SWTBdxFB)
#### Screenshots (if appropriate)
Screen Shot 2020-01-30 at 00 38 02

[finishes #170985787_]

### CODE REVIEW

File: server/db/migrations/20200124175526-create-issues.js [hunk 1]
```diff
@@ -34,11 +34,11 @@ module.exports = {
     },
     issuedDate: {
       allowNull: false,
-      type: Sequelize.DATE,
+      type: Sequelize.STRING,
     },
     returnDate: {
       allowNull: false,
-      type: Sequelize.DATE,
+      type: Sequelize.STRING,
```
Comment (AbdoulNuru) [inline server/db/migrations/20200124175526-create-issues.js:41]: Why did you change the datatypes of issuedDate and returnDate to strings?

Comment (blaise82) [reply server/db/migrations/20200124175526-create-issues.js:41]: if I would have kept to type: Sequelize.DATE, it would automatically add return date 
and I want that to be empty or even a string so that on the return that it can be updated 😞

Comment (AbdoulNuru) [reply server/db/migrations/20200124175526-create-issues.js:41]: @blaise But I think that is not the best way to solve the problem, because you can not explain how a field called issuedDate have string as a datatype.

**Review by AbdoulNuru — APPROVED** [review]

**Review by hakoemmy — APPROVED** [review]

File: server/db/migrations/20200124175526-create-issues.js (hunk 1 above)
Comment (blaise82) [reply server/db/migrations/20200124175526-create-issues.js:41]: @AbdoulNuru  you point is understandable 
if there is a better way to do it and prevent Sequelize from out inputting the date for us even before the book is returned please let me know

**Review by RusimbiPatrick — APPROVED** [review]
